### ISSUE TITLE
Make the LaTeX document

### ISSUE THREAD

**Issue by marc7666** [marc7666/Bachelor-s-thesis#5]

**Comment by marc7666** [comment]
![image](https://user-images.githubusercontent.com/64266158/216642663-658960ba-a140-473d-aba8-819632a67e56.png)

**Comment by marc7666** [comment]
![image](https://user-images.githubusercontent.com/64266158/227455204-395074e1-28ee-48f0-867b-1e9f737442b7.png)

**Comment by marc7666** [comment]
https://github.com/marcotcr/lime

**Comment by marc7666** [comment]
De l'apartat d'anàlisi gràfica falta fer l'anàlisi model a model. Les gràfiques generals estan fetes i comentades.

**Comment by marc7666** [comment]
Apartat "Detalls tècnics d'implementació" => Conceptes que cal remarcar:

- [ ] Definició de batch size
- [ ] Definició de epoch
- [ ] Definició de ImageDataGenerator()
- [ ] Definició de x.flow_from_directory()
- [ ] Definició de model seqüencial
- [ ] Definició de Conv 2D(parametres)
- [ ] Definició de BatchNormalization()
- [ ] Definició de MaxPool2D(parametres)
- [ ] Definició de Dropout()
- [ ] Definició de capa densa
- [ ] Definició de Adam(parametres)
- [ ] Definició de funcio de pèrdua
- [ ] TensorBoard
- [ ] Definició de model.evaluate()
- [ ] Explicar el test simple
- [ ] Explicar apartat Lime (sol codi no imatges)
- [ ] Definició de model.h5

**Comment by marc7666** [comment]
BDCAP = Base de Datos Clínicos de Atención Primaria

**Comment by marc7666** [comment]
![image](https://user-images.githubusercontent.com/64266158/234509888-0cce2010-c38f-4413-8676-4fe0612ace9c.png)

**Comment by marc7666** [comment]
![image](https://user-images.githubusercontent.com/64266158/234509970-f567a7a7-a596-460f-a700-1a7857aaca76.png)

**Comment by marc7666** [comment]
Not explained:
![image](https://github.com/marc7666/Bachelor-s-thesis/assets/64266158/39ef4878-703c-4b41-9c2a-480891389022)

**Comment by marc7666** [comment]
https://keras.io/api/layers/regularization_layers/dropout/
https://vijayabhaskar96.medium.com/tutorial-image-classification-with-keras-flow-from-directory-and-generators-95f75ebe5720
https://keras.io/api/layers/normalization_layers/batch_normalization/
https://keras.io/api/layers/reshaping_layers/flatten/
https://keras.io/api/layers/activations/
